docs/Converters/IXItoBIDS.md

```diff
@@ -0,0 +1,52 @@
+# `ixi-to-bids` – Conversion of Information eXtraction from Images (IXI) to BIDS format
+??? quote "Dataset Description"
+    [IXI webpage](https://brain-development.org/ixi-dataset/)
+    The Information eXtraction from Images is a project which issued a dataset of nearly
+    600 images from healthy subjects. The MR acquisition protocol includes T1,T2, PD weighted,
+    MRA and diffusion-weighted images. Three hospitals in London were involved in data collection.
+
+## Downloading the data
+The IXI dataset can be downloaded freely from the [IXI webpage](https://brain-development.org/ixi-dataset/){ data-preview }.
```

Comment:
What does the `{ data-preview }` part ?

Reply:
I forgot to remove it after testing something ! Will do it now.